src/opera/parser/tosca/v_1_3/parameter_definition.py

```diff
@@ -0,0 +1,32 @@
+from ..bool import Bool
+from ..entity import Entity
+from ..list import List
+from ..map import Map
+from ..reference import Reference
+from ..string import String
+from ..void import Void
+
+
+from .constraint_clause import ConstraintClause
+from .schema_definition import SchemaDefinition
+from .status import Status
+
+
+# TODO(@tadeboro): Unify ParameterDefinition and PropertyDefinition and add
+# runtime checks. Refinement will be a PITA ...
+
+
+class ParameterDefinition(Entity):
+    ATTRS = dict(
+        type=Reference("data_types"),
+        description=String,
+        required=Bool,
+        default=Void,
+        status=Status,
+        constraints=List(ConstraintClause),
+        key_schema=SchemaDefinition,
+        entry_schema=SchemaDefinition,
+        external_schema=String,
+        metadata=Map(String),
```

Comment:
Metadata comes from 3.6.10 since parameter definition is a refinement of property definition. Grammar and keynames sections are not always in sync and in such cases, I assumed keynames to be the correct one.